gsoc_summary.md

```diff
@@ -0,0 +1,82 @@
+# GSOC 2020 Work product submission.
+
+## Summary of GSoC project with conda-forge (NumFocus)
+
+### Introduction
+  Throughout this Google Summer of code ([GSoC](https://summerofcode.withgoogle.com/)) project I, along with my mentors, aimed to improve the conda-forge "auto-tick, by splitting  bot versions update cron job from the main environment, more specifically, separating the version updates structure to its own microservice. Which in return, improved to a more organic bot ecosystem, not only lessening the burden to the original main process, but also granting the bot team a more versatile way to track the packages version datas, and consecutively it's respective issues.
+
+### What is conda-forge
+  [Conda-forge](https://numfocus.org/project/conda-forge), an umbrella project of [NumFOCUS](https://numfocus.org/), is a [community](https://gitter.im/conda-forge/conda-forge.github.io) effort that provides conda packages for a wide range of software, building and distributing packages, specialized in the hard-to-build or unique packages that often arise in a scientific computing context. The conda-forge ecosystem is fully structured around automation, the "auto-tick" bot plays an important role in the automatic maintenance of conda-forge packages, allowing the maintainers to build packages for different systems architectures, as also keeping their packages up-to-date automatically, and ensuring a high level of quality.
+
+
+### How, it works ?
+The conda-forge autotick bot is now a central part of the conda-forge ecosystem. Rougly speaking the bot tracks and updates the feedstocks in four steps:
+   - Find the names of all feedstocks on conda-forge.
+   - Compute the dependency graph of packages on conda-forge found in step 1.
+   - Find the most recent version of each feedstock’s source code.
+   - Open a PR into each out-of-date feedstock updating the meta.yaml for the most recent upstream release.
+   
+  Currently when a package passes by the auto tick bot, it’s information is processed and updated by calling some specific routines, one of these is the version update process. Its main goal is the requisition of every new version for the packages we currently have, getting this information using the sources that are provided for each package in our feedstock, and then load this information at the respective node inside our graph data structure, which then enable the various others services of the bot to update the package, refactor or initane a specific routine, for example a Migrator process.
```

Comment:
initane?

PS: this long line format makes it hard to point the review to the exact part of the text. Usually, for text, the recommended way to facilitate reviews is to break at... Every break! Like commas, dots, etc.

Reply:
good to know